spec/operators/concatAll-spec.ts

```diff
@@ -2,24 +2,33 @@ import { expect } from 'chai';
 import { from, throwError, of, Observable } from 'rxjs';
 import { concatAll, take, mergeMap } from 'rxjs/operators';
 import { TestScheduler } from 'rxjs/testing';
-import { hot, cold, expectObservable, expectSubscriptions } from '../helpers/marble-testing';
+import { observableMatcher } from 'spec/helpers/observableMatcher';
+import { expectObservable } from '../helpers/marble-testing';
```

Comment:
It's likely that the IDE added this automatically (and unnecessarily). We shouldn't need anything from this file once we're using `run`.